src/server/rpc/procedures/google-maps/google-maps.js

```diff
@@ -206,9 +206,39 @@ StaticMap.prototype.getLatitude = function(y){
     });
 };
 
+/**
+ * converts image coordinates to earth coordinates.
```

Comment:
Rephrase "Get the given earth coordinates (latitude, longitude) of a given point in the last requested map image (x, y)"